src/service/idam-api/src/FamilyHubs.Idam.Core/Commands/Delete/DeleteExpiredUserSessionsCommand.cs

```diff
@@ -31,19 +28,19 @@ public async Task<bool> Handle(DeleteExpiredUserSessionsCommand request, Cancell
         {
             _logger.LogInformation("DeleteExpiredUserSessionsCommand started");
 
-            int? cleanupInterval = _configuration.GetValue<int?>("ExpiredSessionCleanupInterval");
+            var cleanupInterval = _configuration.GetValue<int?>("ExpiredSessionCleanupInterval");
             if (cleanupInterval is null)
             {
                 _logger.LogError("ExpiredSessionCleanupInterval not configured.");
                 throw new IdamsException("ExpiredSessionCleanupInterval not configured.");
             }
-            DateTime sessionExpiryTime = DateTime.UtcNow.AddSeconds((int)-cleanupInterval);
-            List<UserSession> entities =
+            var sessionExpiryTime = DateTime.UtcNow.AddSeconds((int)-cleanupInterval);
+            var entities =
                 await _dbContext.UserSessions.Where(r => r.LastActive < sessionExpiryTime).ToListAsync(cancellationToken);
 
             _logger.LogInformation("Found: {CountExpired} expired sessions.", entities.Count);
 
-            if (entities.Any())
+            if (entities.Count != 0)
```

Comment:
Probably > as Count will never be less than zero

Reply:
The != 0 is the direct suggested replacement of Any() but yes that also makes sense